src/mesh/interpolation_xz.cxx

```diff
@@ -38,7 +38,9 @@ const Field3D interpolate(const Field3D& f, const Field3D& delta_x,
                           const Field3D& delta_z) {
   TRACE("Interpolating 3D field");
   XZLagrange4pt interpolateMethod{f.getMesh()};
-  return interpolateMethod.interpolate(f, delta_x, delta_z);
+  // Cast to base pointer so virtual function overload is resolved
+  return static_cast<XZInterpolation*>(&interpolateMethod)
+      ->interpolate(f, delta_x, delta_z);
```

Comment:
This shouldn't be needed -- the issue is probably because we now need a `using XZInterpolation::interpolate` in the derived classes to bring in the missing overloads of the virtual function.